tensorflow/python/ops/nn_ops.py

```diff
@@ -5334,6 +5334,8 @@ def fractional_max_pool_v2(value,
       [Graham, 2015](https://arxiv.org/abs/1412.6071)
       ([pdf](https://arxiv.org/pdf/1412.6071.pdf))
   """
+  if pooling_ratio < 1.0:
```

Comment:
pooling_ration can be a list, in which case this doesn't work.

Reply:
Please add tests.

Reply:
@rmlarsen  I have added tests for list also.